docs/core-concepts/dotnet-core.md

```diff
@@ -0,0 +1,63 @@
+
+The C#, F# and VB compilers all support targeting and running on .NET Core. .NET Core tools are integrated into several text editors and IDEs, including Visual Studio, Visual Studio Code, Sublime Text and Vim, providing .NET Core development experiences in your favorite environment. 
+
+.NET Core is open source (MIT license) and was contributed to the [.NET Foundation](http://dotnetfoundation.org) by Microsoft in 2014 and is now one of the most active .NET Foundation projects. It can be freely adopted by individuals and companies, including for personal, academic or commercial purposes. Multiple companies are using .NET Core as part of apps, tools, new platforms and hosting services. Some of these companies make significant contributions to .NET Core on GitHub and provide guidance on the product direction as part of the [.NET Foundation Technical Steering Group](http://www.dotnetfoundation.org/blog/tsg-welcome).
+
+.NET Core v1 supports the following workloads and app-models:
+
+- Class Libraries
+- Console apps
+- ASP.NET Core websites and services
+- UWP Windows 10 apps
+
+You can get started with .NET Core in just a few minutes:
+
+- [Getting started with .NET Core](https://aka.ms/dotnetcoregs)
+- [.NET Core samples](https://github.com/dotnet/core)
+- .NET Core OS and CPU Support
+- [Explore .NET Core repos](https://github.com/dotnet/core)
+
+Composition of .NET Core
+------------------------
+
+**.NET Core is the following:**
+
+- **A set of packages** -- These contain the runtime and the API shape and implementation of .NET Core.
+- **A set of metapackages (for example, `NETStandard.Library`)** --  They describe various layers of .NET Core by referencing the appropriate set of library packages.
+- **A [distribution](https://aka.ms/dotnetcoregs) of .NET Core from Microsoft** -- This includes the CoreCLR runtime, associated libraries, the console application host and the `dotnet` launcher. It is described by the `Microsoft.NETCore.App` metapackage.
+- **A [distribution](https://dotnet.github.io) of the .NET Core SDK from Microsoft** -- This includes a set of tools for restoring NuGet packages and compiling and building apps, also controlled from the `dotnet` launcher. It is extensible, enabling anyone to add commands. The package also includes a copy of .NET Core to use with the tools. Typically, developers will install this distribution.
+
+**Closely affiliated with .NET Core:**
+
+- **The ".NET Standard Library"** - This is the API spec that describes the API evolution of .NET and that .NET Core implements. While many of the additions to the Standard Library will come from API additions to .NET Core, some will come from other platforms, such as the .NET Framework and Mono.
+- **System.* packages** - These packages provide the ".NET Core" implementation but contain assets for other platforms, such as the .NET Framework and Mono.
```

Comment:
Nit: Lists above and below use `--`, this list uses `-`. I think all lists should use `-` to be consistent (or &ndash;, `&ndash;`, but that's hard to type).